src/test/java/org/apache/maven/plugins/dependency/fromDependencies/TestCopyDependenciesMojo.java

```diff
@@ -80,7 +80,7 @@ public void assertNoMarkerFile(Artifact artifact) throws MojoExecutionException
         assertFalse(handle.isMarkerSet());
     }
 
-    public void testCopyFile() throws MojoExecutionException, IOException {
+    public void testCopyFile() throws Exception {
```

Comment:
did this change to throw a raw exception? Otherwise it's better to declare the actual exceptions instead of the common superclass.

Reply:
It is a Unit Test ... all other methods simply throws `Exception`

Reply:
I like to know when an exception is added or removed from an API, and I'd make the same comment about the other methods in this class. :-)

Reply:
ok, one exception is new in 3.7.0 but one is old from copied  class - so should I discovered versions for original class or add since now